hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioNetworking.java

```diff
@@ -157,17 +163,12 @@ public void start() {
 
         logger.log(selectorMode != SELECT ? Level.INFO : FINE, "IO threads selector mode is " + selectorMode);
 
-        if (metricsRegistryScheduled.compareAndSet(false, true) && metricsRegistry.minimumLevel().isEnabled(ProbeLevel.DEBUG)) {
-            metricsRegistry.scheduleAtFixedRate(new PublishAllTask(), 1, SECONDS, ProbeLevel.INFO);
-        }
+        publishFuture = metricsRegistry.scheduleAtFixedRate(new PublishAllTask(), 1, SECONDS, ProbeLevel.INFO);
```

Comment:
What's with the missing `metricsRegistry.minimumLevel().isEnabled(ProbeLevel.DEBUG)` condition?

Reply:
Ah, I see that is always `true`.